tensorflow_probability/python/distributions/laplace.py

```diff
@@ -202,6 +202,9 @@ def _median(self):
   def _mode(self):
     return self._mean()
 
+  def _quantile(self, x):
```

Comment:
You should rename input from `x` to `p` for consistency with the rest of the distributions.

Reply:
Done.